addOns/callhome/src/main/java/org/zaproxy/addon/callhome/ExtensionCallHome.java

```diff
@@ -90,17 +111,41 @@ public String toString() {
     }
 
     private HttpSender httpSender = null;
+    private CallHomeParam param;
 
     private static OS os;
     private static Boolean onBackBox = null;
     private static Boolean inContainer = null;
     private static String containerName;
 
+    private int telIndex = 0;
+    private long startMs = System.currentTimeMillis();
```

Comment:
Could use `RuntimeMXBean` which should be more accurate (e.g. if the add-on is updated this would result in a different uptime).